src/policy/space.rs

```diff
@@ -181,8 +181,9 @@ impl SFT for EmptySpaceSFT {
         _object: ObjectReference,
         _worker: GCWorkerMutRef,
     ) -> ObjectReference {
+        // We do not have the `VM` type parameter here, so we cannot forward the call to the VM.
         panic!(
-            "Call trace_object() on {} (chunk {}), which maps to an empty space",
+            "Call trace_object() on {} (chunk {}), which maps to an empty space. SFTProcessEdges does not support the fallback to vm_trace_object().",
             _object,
```

Comment:
This `_object` has the same problem.  Although it is not related to the bindings, it is nice to fix it, too.